pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerBuilder.java

```diff
@@ -164,37 +164,37 @@
     ProducerBuilder<T> sendTimeout(int sendTimeout, TimeUnit unit);
 
     /**
-     * Set the max size of the queue holding the messages pending to receive an acknowledgment from the broker.
+     * Set the max size of the messages pending queue to receive an acknowledgment from the broker.
```

Comment:
```suggestion
     * Set the max size of the queue holding the messages pending to receive an acknowledgment from the broker.
```

Revert. Not quite different.